_topic_maps/_topic_map.yml

```diff
@@ -365,6 +365,15 @@ Topics:
     File: serverless-logic-configuring-openAPI-services-endpoints
   - Name: Troubleshooting services
     File: serverless-logic-troubleshooting-services
+- Name: Supporting services
+  Dir: serverless-logic-supporting-services
```

Comment:
IMO, this is a source of error. If you specify a `Dir`, then it expects at least one `File` to be listed under it with an indent.